doc/source/serve/doc_code/vllm_openai_example.py

```diff
@@ -116,6 +124,20 @@ def build_app(cli_args: Dict[str, str]) -> serve.Application:
 
     Supported engine arguments: https://docs.vllm.ai/en/latest/models/engine_args.html.
     """  # noqa: E501
+    if "accelerator" in cli_args.keys():
+        accelerator = cli_args.pop("accelerator")
+    else:
+        accelerator = "GPU"
+    if accelerator == "HPU":
+        try:
+            from habana_frameworks.torch.distributed.hccl import (
+                initialize_distributed_hpu,
+            )
+
+            initialize_distributed_hpu()
```

Comment:
what does this do? Is the assumption that this is being run on a node with HPU?

Reply:
This is to confirm HPU environment, if we assume that user is running code on the correct device, I would remove this check.

Reply:
the application builder may not run on a node that has HPU (eg. the head node might be cpu only)

Reply:
Got it. I've removed it.